src/bun.js/api/bun/socket/Handlers.zig

```diff
@@ -98,7 +98,7 @@ pub fn callErrorHandler(this: *Handlers, thisValue: JSValue, args: *const [2]JSV
         return false;
     }
 
-    _ = onError.call(globalObject, thisValue, args) catch |e| globalObject.reportActiveExceptionAsUnhandled(e);
+    _ = onError.call(globalObject, thisValue, args) catch |e| try globalObject.reportActiveExceptionAsUnhandled(e);
```

Comment:
can we extract run callback and report error as an uncaught exception but return js termination exception into globalObject to make it more clear what this is doing?